tools/remote_build/windows.bazelrc

```diff
@@ -1,3 +1,51 @@
 # TODO(yfen): Merge with rbe_common.bazelrc and enable Windows RBE
+
```

Comment:
nit: the README.md file in this directory still says that this runs a "local" build, which is no longer true.

Reply:
addressed